platform/rpc/nats/bus_test.go

```diff
@@ -0,0 +1,190 @@
+package nats
```

Comment:
It would be nice if we didn't *need* nats running to run the tests. Maybe a build flag: `// +build integration` ? I don't feel strongly about it, though, wdyt?